ts/src/index.ts

```diff
@@ -153,22 +153,60 @@ function logError(msg: string, config: Config) {
  */
 export async function startLocal(config: Config = {}): Promise<void> {
   const normalizedConfig = await initConfig(config);
-  profiler = new Profiler(normalizedConfig);
-
-  while (true) {
+  const profiler = new Profiler(normalizedConfig);
+
+  // Set up periodic logging.
+  const logger = new common.logger({
+    level: common.logger.LEVELS[normalizedConfig.logLevel],
+    tag: pjson.name
+  });
+  let heapProfileCount = 0;
+  let timeProfileCount = 0;
+  let prevLogTime = Date.now();
+
+  setInterval(() => {
+    const curTime = Date.now();
+    const {rss, heapTotal, heapUsed} = process.memoryUsage();
+    logger.debug(
```

Comment:
What the example output looks like? It appears there are no any separators between metric name and value. Want to make sure it looks readable.

Reply:
Sample output:

```
DEBUG:@google-cloud/profiler: 2018-04-18T16:47:00.579Z rss 97.598 MiB, heap total 62.840 MiB, heap used 25.339 MiB, heap profile collection rate 338.900 profiles/s time profile collection rate 0.000 profiles/s
DEBUG:@google-cloud/profiler: 2018-04-18T16:47:10.580Z rss 98.879 MiB, heap total 63.840 MiB, heap used 22.269 MiB, heap profile collection rate 356.800 profiles/s time profile collection rate 0.000 profiles/s
DEBUG:@google-cloud/profiler: 2018-04-18T16:47:20.581Z rss 98.215 MiB, heap total 63.340 MiB, heap used 25.012 MiB, heap profile collection rate 355.329 profiles/s time profile collection rate 0.000 profiles/s
```

Reply:
OK. Is there a comma missing after first "profiles/s"?

Reply:
Looks like it was done.